component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticatorConstants.java

```diff
@@ -172,7 +172,7 @@ private EmailOTPAuthenticatorConstants() {
     public static final String IS_IDF_INITIATED_FROM_AUTHENTICATOR = "isIdfInitiatedFromAuthenticator";
     public static final String IDF_HANDLER_NAME = "IdentifierExecutor";
     public static final String USER = "user";
-    public static final String EMAIL_OTP_NUMERIC_OTP = "OnlyNumericCharactersForOtp";
```

Comment:
We cannot get rid of the public constants like this? What if this constant is being used anywhere else?

Reply:
This is a constant used to refer to the config name and as it is a public variable, if you need to refer to it you can import the variable from here.

Reply:
This constant is introduced after 6.1 release.